modules/create-policy-from-system-policies-view.adoc

```diff
@@ -8,58 +8,67 @@
+In other clusters, {product-title-short} edits noncompliant deployments to prevent pods from being scheduled.
+**** *Runtime* - {product-title-short} kills all pods when an event in the pods matches the criteria of the policy.
 +
 [WARNING]
 ====
 Policy enforcement can impact running applications or development processes.
 Before you enable enforcement options, inform all stakeholders and plan about how to respond to automated enforcement actions.
 ====
 . Click *Next*.
-. In the *Policy Criteria* section, configure the attributes that you you want to trigger the policy for.
+. In the *Policy Criteria* section, configure the attributes that you want to trigger the policy for.
+.. Click and drag policy fields into the *Policy Section* to add criteria.
++
+[NOTE]
+====
+The policy fields that are available depend on the lifecycle stage you chose for the policy. For example, criteria under `Kubernetes access policies` or `Networking` are available when creating a policy for the runtime lifecycle, but not when creating a policy for the build lifecycle. See "Policy criteria" in the "Additional resources" section for more information about policy criteria, including information about criteria and the lifecycle phase in which they are available.
+====
+.. Optional: Click *Add condition* to add policy sections that contain additional criteria that will trigger the policy (for example, to trigger on old, stale images, you can configure that `image tag` *is not* `latest` *or* `image age` and specify a minimum number of days since an image is built).
 . Click *Next*.
 . In the *Policy scope* section, configure the following:
-** Click *Add inclusion scope* to use *Restrict to Scope* to enable this policy only for a specific cluster, a namespace, or a label.
-You can add multiple scopes and also use regular expressions in link:https://github.com/google/re2/wiki/Syntax[RE2 Syntax] for namespaces and labels.
-** Click *Add exclusion scope* to use *Exclude by Scope* to exclude deployments, clusters, namespaces, and labels you specify, it means that the policy will not apply to the entities that you select.
-You can add multiple scopes and also use regular expressions in link:https://github.com/google/re2/wiki/Syntax[RE2 Syntax] for namespaces and labels.
-However, you cannot use regular expressions for selecting deployments.
+** Click *Add inclusion scope* to use *Restrict by scope* to enable this policy only for a specific cluster, a namespace, or a label.
+You can add multiple scopes and also use regular expression in link:https://github.com/google/re2/wiki/Syntax[RE2 Syntax] for namespaces and labels.
+** Click *Add exclusion scope* to use *Exclude by scope* to exclude deployments, clusters, namespaces, and labels you specify. The policy will not apply to the entities that you select.
+You can add multiple scopes and also use regular expression in link:https://github.com/google/re2/wiki/Syntax[RE2 Syntax] for namespaces and labels.
```

Comment:
Should the two instances (line 70 and 71) of "regular expression" be "regular expressions"?

Reply:
I've seen both, but I actually researched this for the RH SSG and we went with the singular form. I guess if I were talking about entering multiple regex items, then the plural would be more accurate, but I think this usage is OK.